Compentio.SourceMapper.Generator/Generators/CodeSourceGenerator.cs

```diff
@@ -20,25 +21,25 @@ internal class CodeSourceGenerator
         private readonly ISourcesMetadata _sourcesMetadata;
         private readonly List<DiagnosticsInfo> _diagnostics = new();
 
-        internal CodeSourceGenerator (ISourcesMetadata sourcesMetadata)
+        internal CodeSourceGenerator(ISourcesMetadata sourcesMetadata)
         {
             _sourcesMetadata = sourcesMetadata;
         }
 
         internal void GenerateMappings(GeneratorExecutionContext context)
         {
-            foreach (var mapper in _sourcesMetadata.Mappers)
+            foreach (var mapper in _sourcesMetadata.Mappers.Where(m => !m.IsExternal))
```

Comment:
Do we generate code for external mappers or now?

Reply:
It can be confused - I changes it to IsReferenced. In this place we skip generating duplication of mappers from referenced projects/dlls.